geode-core/src/main/java/org/apache/geode/management/internal/ManagementAgent.java

```diff
@@ -328,6 +329,40 @@ private void startHttpService(boolean isServer) {
     }
   }
 
+  private Properties createSslProps() {
```

Comment:
I think, because it's so specific to Pulse, it's better to be closer to where it's relevant. Also, because it's referring to 'rmi' client specific props.